OverlayPlugin.Core/MemoryProcessors/FFXIVMemory.cs

```diff
@@ -27,40 +28,45 @@ public FFXIVMemory(TinyIoCContainer container)
 
         private void UpdateProcess(Process proc)
         {
-            if (processHandle != IntPtr.Zero)
+            lock (_mutex)
```

Comment:
It's not good to `MessageBox.Show` in mutex because `MessageBox.Show` is a user-blocking function.